🐛 [Bug]:  生成代码报错

### Environment

windwos11

### Version

16

### Version

最新

### Link to minimal reproduction

Cannot find module 'C:\Users\cq\Desktop\12\tiny-engine-develop\tiny-engine-develop\mockServer\node_modules\@opentiny\tiny-engine-dsl-vue\dist\tiny-engine-dsl-vue.cjs.js'. Please verify that the package.json has a valid "main" entry

### Step to reproduce

点击生成

### What is expected

_No response_

### What is actually happening

_No response_

### Any additional comments (optional)

_No response_

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "preinstall": "npx only-allow pnpm",
-    "dev": "concurrently 'pnpm:serve:backend' 'pnpm:serve:frontend'",
+    "dev": "pnpm run setup && concurrently 'pnpm:serve:backend' 'pnpm:serve:frontend'",
     "dev:mock": "pnpm --filter @opentiny/tiny-engine dev",
     "serve:frontend": "pnpm --filter @opentiny/tiny-engine serve",
     "serve:backend": "pnpm --filter @opentiny/tiny-engine-mock dev",
@@ -18,7 +18,8 @@
     "pub:premajor": "pnpm run build:plugin && pnpm run build:alpha && pnpm lerna version premajor --preid beta --no-push --yes && lerna publish from-package --pre-dist-tag beta --yes",
     "pub:preminor": "pnpm run build:plugin && pnpm run build:alpha && pnpm lerna version preminor --preid beta --no-push --yes && lerna publish from-package --pre-dist-tag beta --yes",
     "pub:prepatch": "pnpm run build:plugin && pnpm run build:alpha && pnpm lerna version prepatch --preid beta --no-push --yes && lerna publish from-package --pre-dist-tag beta --yes",
-    "pub:prerelease": "pnpm run build:plugin && pnpm run build:alpha && pnpm lerna version prerelease --preid beta --no-push --yes && lerna publish from-package --pre-dist-tag beta --yes"
+    "pub:prerelease": "pnpm run build:plugin && pnpm run build:alpha && pnpm lerna version prerelease --preid beta --no-push --yes && lerna publish from-package --pre-dist-tag beta --yes",
+    "setup": "node ./scripts/setup.js"
   },
   "devDependencies": {
     "@babel/eslint-parser": "^7.21.3",

--- a/scripts/setup.js
+++ b/scripts/setup.js
@@ -1,0 +1,9 @@
+const { exec } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+fs.access(path.resolve(__dirname, 'packages/vue-generator/dist'), (err) => {
+  if (err) {
+    exec('pnpm -F @opentiny/tiny-engine-dsl-vue build')
+  }
+})

